Wikipedia/Code/WMFExploreViewController.m

```diff
@@ -154,6 +154,10 @@ - (WMFExploreSectionControllerCache*)sectionControllerCache {
     return _sectionControllerCache;
 }
 
+- (NSUInteger)numberOfSectionsInExploreFeed{
```

Comment:
I think we prob should? To ease future Swift inter-op and/or conversion to Swift...